temporalio/workflow.py

```diff
@@ -173,6 +173,20 @@ def run(fn: CallableAsyncType) -> CallableAsyncType:
     return fn  # type: ignore[return-value]
 
 
+class UnfinishedHandlersPolicy(IntEnum):
```

Comment:
```suggestion
class UnfinishedHandlersPolicy(Enum):
```

I don't think it needs to extend int for our use